src/pages/home/_what-our-clients-say.tsx

```diff
@@ -8,6 +8,14 @@ import device from 'themes/device'
 import Quote from 'images/svg/testimonials/quote.svg'
 import { useCountryRule } from 'components/hooks/use-country-rule'
 
+declare global {
```

Comment:
why do we need to do this?

Reply:
Yes @eduard-deriv , we don't have this in the `global this.window` so it throw an error, for fixing that I defined this type